core/services/keeper/registry_synchronizer_core.go

```diff
@@ -108,7 +109,7 @@ func (rs *RegistrySynchronizer) Close() error {
 	return rs.StopOnce("RegistrySynchronizer", func() error {
 		close(rs.chStop)
 		rs.wgDone.Wait()
-		return nil
+		return rs.mbLogs.Close()
```

Comment:
Was this leaking before?

Reply:
I don't think so. AFAIK channels do not need to be closed to be garbage collected. Leaving them in the monitor map would keep them alive though.